src/test/java/bio/terra/service/auth/iam/sam/SamIamTest.java

```diff
@@ -867,4 +868,37 @@ void testDeleteGroupThrowsWhenSamGroupApiThrows() throws ApiException {
       assertThrows(IamNotFoundException.class, () -> samIam.deleteGroup(accessToken, groupName));
     }
   }
+
+  @Nested
```

Comment:
Oh neat, I haven't previously worked with JUnit 5's nested tests, this is a nice way to facilitate the set-up common to this group!